tests/test_pipeline.py

```diff
@@ -364,8 +364,7 @@ def test_replicate_pipeline_from_config(self):
                     model="transe",
                 ),
                 results={
-                    "hits_at_k": {"best": {"10": 0.538}},
-                    "mean_rank": {"best": 163},
+                    "best": {"hits_at_k": {"10": 0.538}, "mean_rank": 163},
```

Comment:
@cthoyt this may imply that we need to update the structure of the provided metrics in the experimental configurations, too